hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -107,6 +107,58 @@
       '''
     }
   ],
+  features: [
+    { name: "SPIHOST.USECASE.SERIALNORFLASH",
+      desc: '''
+        The SPI host block should be able to talk to serial NOR flash devices.
+      '''
+    },
+    { name: "SPIHOST.USECASE.PASSTHROUGH",
+      desc: '''
+        The SPI host block should be able to talk to serial NOR flash devices.
+      '''
+    },
+    { name: "SPIHOST.RATE.STANDARD",
+      desc: '''
+        Standard SPI data rate.
+      '''
+    },
+    { name: "SPIHOST.RATE.DUAL",
+      desc: '''
+        Dual SPI data rate.
+      '''
+    },
+    { name: "SPIHOST.RATE.QUAD",
+      desc: '''
+        Quad SPI data rate.
+      '''
+    },
+    { name: "SPIHOST.CONFIG.CPOL",
+      desc: '''
+        Polarity of the SPI.
```

Comment:
If we're describing a feature, maybe it's that the polarity of the SPI can be configured? (After all, any implementation will have *some* polarity!)

Reply:
Thanks, I've reworded this to hopefully address this.